products/openzfs.md

```diff
@@ -26,11 +26,19 @@ auto:
 # LTS: eol(x) = estimation: releaseDate(x) plus 2 years
 # supportedLinux / supportedFreeBSD is available at the top of each release note and evolve even in minor versions.
 releases:
+-   releaseCycle: "2.3"
+    supportedLinux: "4.18 - 6.12"
+    supportedFreeBSD: "13.3-RELEASE+"
+    releaseDate: 2025-01-13
+    eol: false # releaseDate(2.4)
+    latest: "2.3.0"
+    latestReleaseDate: 2025-01-13
+
 -   releaseCycle: "2.2"
-    supportedLinux: "4.18 - 6.10"
-    supportedFreeBSD: "12.2-RELEASE+"
+    supportedLinux: "4.18 - 6.12"
+    supportedFreeBSD: "13.0-RELEASE+"
```

Comment:
is it dropped support for freebsd 12.2 ?

Reply:
looks like a mistake there but not sure , could you double check ? I mean if it was supporting they never drop while in a minor update

Reply:
yeah, it dropped per this PR https://github.com/openzfs/zfs/pull/16372